src/test/resources/mappings/search-definition-1.json

```diff
@@ -1,7 +1,7 @@
 {
   "request": {
     "method": "GET",
-    "url": "/api/display/search-result-definition/TestAddressBookCase?version=33"
+    "urlPath": "/api/display/search-result-definition/TestAddressBookCase"
```

Comment:
out of curiosity how come this change?

Reply:
so that i can control different version with in tests
Eg:  https://github.com/hmcts/ccd-data-store-api/blob/RDM-3974/src/test/java/uk/gov/hmcts/ccd/endpoint/ui/SearchWithSortIT.java#L71

Reply:
👍